packages/api/controllers/auth.js

```diff
@@ -80,36 +75,41 @@ const markAsRead = function (messageId, gmail) {
     }
   });
 };
-
 async function getNewDocs(auth) {
   const gmail = google.gmail({ version: 'v1', auth });
-  let query = '';
+  const query = '';
   const nomNames = await getAwaitingHipaa();
-
-  if (!nomNames) {
-    console.log('No Nominations Currently Awaiting HIPAA');
-  } else {
-    nomNames.forEach((nomination) => query += `subject:'${nomination}'`);
-    query += 'AND is:unread';
-    gmail.users.messages.list({
-      userId: 'me',
-      q: `{${query}}`,
-    }, (err, res) => {
-      if (err) return console.log('The API returned an error: ', err);
-      const { messages } = res.data;
-      if (!messages) {
-        console.log('no messages');
-      } else if (messages.length) {
-        const messageIds = [];
-        messagesAndAttachments(messages, messageIds, gmail, nomNames);
-      }
-    });
+  const receiptNoms = await getVerifiedNoms();
+  if (Object.entries(nomNames).length) {
```

Comment:
Is nomNames an array? If so, you can just check `if (nomNames?.length)`

Reply:
this nomNames is an Object.

Reply:
Okay cool